<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Alexanders-MacBook-Pro.local" ignored="0" name="check log" tests="1" failures="1" timestamp="31 Mar 2020 11:33:02 GMT" time="3.62" errors="0">
  <testcase name="checkLog" time="1.707" classname="tests.JsErrorTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Failed because of JS error: http://the-internet.herokuapp.com/favicon.ico - Failed to load resource: the server responded with a status of 404 (Not Found),
	Failed because of JS error: http://the-internet.herokuapp.com/javascript_error 6:51 Uncaught TypeError: Cannot read property &amp;apos;xyz&amp;apos; of undefined">
      <![CDATA[java.lang.AssertionError: The following asserts failed:Failed because of JS error: http://the-internet.herokuapp.com/favicon.ico - Failed to load resource: the server responded with a status of 404 (Not Found),Failed because of JS error: http://the-internet.herokuapp.com/javascript_error 6:51 Uncaught TypeError: Cannot read property 'xyz' of undefinedat tests.JsErrorTest.checkLog(JsErrorTest.java:49)... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- checkLog -->
</testsuite> <!-- check log -->
